TechCommunityBlogs/ignite-highlight-21.md

```diff
@@ -0,0 +1,87 @@
+
+# Microsoft Ignite Highlights: Azure CLI
+
+Hi everyone! as Microsoft Ignite wraps up, we’re sharing some of the latest features we released in
+the Azure CLI supporting various announcements for Microsoft Ignite. We also released several
+important updates to Azure CLI commands and even a new extension. Here are some of the exciting
+announcements and updates.
+
+(NOTE: The Azure CLI commands in the format az command --parameter that are shown below are not
+intended to be run as is, they only show the key parameters that need to be used for the feature
+described. Please refer to the documentation links provided to learn how to use the command with all
+the required parameters.)
+
+## Compute
+
+**Azure VMSS flexible orchestration mode support**. With the GA release of [Azure VMSS flexible orchestration mode](https://docs.microsoft.com/azure/virtual-machines/flexible-virtual-machine-scale-sets),
+Azure CLI has released several updates to support the release including updated default values for
+an easier VMSS Flex creation experience and user friendly error messaging to help identify
+limitations when creating scale sets with flexible orchestration.
+
+**New `az vm/vmss --run-command` commands**. You now have more control over your [run-commands](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/run-command)
```

Comment:
Could we change the title to something like this?
> Added new commands to enhance the support for VM/VMSS run-command

This is because some of the basic commands have existed before, and we had added a series of more perfect commands on this basis in ignite. Therefore, this description may be more accurate

Reply:
Good point. I've update the new highlight. Thanks!